### PR TITLE
icon components

### PR DESCRIPTION
**Pull request by ainneo**
Icon components have been created and ready for review

### CODE REVIEW

**Comment by ExperimentsInHonesty** [comment]
Fixes #226

File: src/components/common/Icons.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+export default Icons;
+
+Icons.propTypes = {
+  id: PropTypes.string.isRequired,
+  label: PropTypes.string,
+  handleClick: PropTypes.func,
+  color: PropTypes.string,
+  size: PropTypes.oneOf(['small', 'medium', 'large']),
+  iconTag:PropTypes.string,
+  iconSize:PropTypes.string,
```
Comment (brodly) [inline src/components/common/Icons.js]: Do you think we can add a PropTypes.oneOf here similar to how you did the size proptype, but with the FontAwesome size variations?

File: src/components/common/Icons.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+export default Icons;
+
+Icons.propTypes = {
+  id: PropTypes.string.isRequired,
+  label: PropTypes.string,
+  handleClick: PropTypes.func,
+  color: PropTypes.string,
+  size: PropTypes.oneOf(['small', 'medium', 'large']),
+  iconTag:PropTypes.string,
```
Comment (brodly) [inline src/components/common/Icons.js]: Can we change the word `iconTag` to `icon`?

File: src/components/common/Icons.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  */
+  color,
+  size,
+}) => {
+  // Dynamically generates infoIcon className from props to comply with Bulma styling modifiers.
+  const infoClassName = classNames('icon', {
+    [`is-${size}`]: size, //for small, meduim, large, span tag
+    [`has-text-${color}`]: color,
+    [`fas-fa${iconSize}`]:iconSize //for fa-lg, fa-2x, fa-3x, icon tag
+  });
```
Comment (brodly) [inline src/components/common/Icons.js]: add `icon` to the classNames function so we can just add the name of the icon without the `fa-`

File: src/components/common/Icons.js (hunk 1 above)
Comment (adamkendis) [reply src/components/common/Icons.js]: Added propTypes options for iconSize.

File: src/components/common/Icons.js (hunk 2 above)
Comment (adamkendis) [reply src/components/common/Icons.js]: Changed as requested.

File: src/components/common/Icons.js (hunk 3 above)
Comment (adamkendis) [reply src/components/common/Icons.js]: Completed.

**Pull request merged**